packages/server/src/fhir/search.test.ts

```diff
@@ -4234,7 +4236,9 @@ describe('FHIR Search', () => {
         expect(searchResult1.entry?.[0]?.resource?.id).toBe(bundle.id);
 
         // Chained search on the Composition
-        const searchResult2 = await repo.search(parseSearchRequest(`Bundle?composition.date=2025-01-01`));
```

Comment:
At present, searching on a `dateTime` field with a `date` string can lead to some funky results.

In the future, when we move to proper intervals for everything, then this would (insert time/zone caveats) work as expected.

For now, best to be explicit for deterministic results.